library/kani_core/src/lib.rs

```diff
@@ -58,6 +59,11 @@ macro_rules! kani_lib {
         kani_core::kani_intrinsics!(std);
         kani_core::generate_arbitrary!(std);
 
+        kani_core::check_intrinsic! {
+        msg="This API was accidently added as a public function and it will be made private in future releases.",
```

Comment:
Can you please create a tracking issue with a proposed timeline so that we don't forget to act upon this?

Reply:
https://github.com/model-checking/kani/issues/3561